jwt-authorizer/tests/integration_tests.rs

```diff
@@ -77,11 +76,8 @@ fn run_jwks_server() -> String {
         .route("/jwks", get(jwks));
 
     tokio::spawn(async move {
-        axum::Server::from_tcp(listener)
-            .unwrap()
-            .serve(app.into_make_service())
-            .await
-            .unwrap();
+        let listener: tokio::net::TcpListener = tokio::net::TcpListener::from_std(listener).unwrap();
```

Comment:
Because now won't wrap `std::net::TcpListener` into `tokio::net::TcpListener` in 0.7, issues arise. Is there any issue with `run_jwks_server()` becoming async? More importantly, since this is a critical component in your integration tests, is there any reason why you want to use `std::net::TcpListener` over `tokio::net::TcpListener`?

Reply:
Resolved by making the `std` listener non-blocking